README.md

```diff
@@ -112,7 +112,6 @@ tkn pipeline start operator-hosted-pipeline \
   --workspace name=pyxis-ssl-credentials,secret=operator-pipeline-api-certs \
   --workspace name=kubeconfig,secret=kubeconfig \
   --workspace name=github-bot-token,secret=github-bot-token \
-  --workspace name=kubeconfig,secret=kubeconfig \
```

Comment:
Hey! Are you sure about the removal of that from there? As it still is in the pipeline definition, comments, ansible

Reply:
This is just removal of a duplicate workspace also provided two lines above.